tests/timeseries/_battery_pool/test_battery_pool_control_methods.py

```diff
@@ -150,14 +151,9 @@ def _assert_report(
             Power.from_watts(power) if power is not None else None
         )
         # pylint: disable=protected-access
-        assert (
-            report._inclusion_bounds is not None
-            and report._exclusion_bounds is not None
-        )
-        assert report._inclusion_bounds.lower == Power.from_watts(lower)
-        assert report._inclusion_bounds.upper == Power.from_watts(upper)
-        assert report._exclusion_bounds.lower == Power.from_watts(0.0)
-        assert report._exclusion_bounds.upper == Power.from_watts(0.0)
+        assert report.bounds is not None
+        assert report.bounds.lower == Power.from_watts(lower)
+        assert report.bounds.upper == Power.from_watts(upper)
         # pylint: enable=protected-access
```

Comment:
disable/enable protected-access might be no longer needed

Reply:
done: https://github.com/frequenz-floss/frequenz-sdk-python/commit/91a3c825